### PR TITLE
article creation

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-dd89d666da/how-to/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-ba3beb691a/how-to/)

File: content/rackspace-email-archiving/log-in-to-the-archive-manager-for-office-365.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+permalink: log-in-to-the-archive-manager-for-office-365/
+audit_date:
+title: Log in to the Archive Manager for Office 365
+type: article
+created_date: '2018-01-03'
+created_by: Brittany Gonzales
+last_modified_date: '2018-01-03'
```
Comment (cyrichardson) [inline content/rackspace-email-archiving/log-in-to-the-archive-manager-for-office-365.md]: Do you want to change the last_modified_date?

File: content/rackspace-email-archiving/log-in-to-the-archive-manager-for-office-365.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+For more information on prerequisite terminology, see [Cloud Office support terminology](/how-to/cloud-office-support-terminology).
+
+
+### Log in to the archive manager
+
+You can log in to the Email Archive Manager directly from your
+domain-specific URL (for example, `http://yourdomaincom.archivesrvr.com`).
+
+Alternatively, you can log in through the Cloud Office Control Panel by
+performing the following steps:
```
Comment (cyrichardson) [inline content/rackspace-email-archiving/log-in-to-the-archive-manager-for-office-365.md]: using the following steps:

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-e0c91db2fa/how-to/)

**Pull request merged**